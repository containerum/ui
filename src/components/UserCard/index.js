/* @flow */

import React from 'react';

type Props = { info: Object };

const UserCard = ({ info }: Props) => (
  <div>
    <h4>User Card</h4>
    <ul>
      <li>Name: {info.name}</li>
      <li>Phone: {info.phone}</li>
      <li>Email: {info.email}</li>
      <li>Website: {info.website}</li>
    </ul>
  </div>
);

export default UserCard;
