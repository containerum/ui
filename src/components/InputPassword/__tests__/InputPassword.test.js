import React from 'react';
import renderer from 'react-test-renderer';
import { StaticRouter } from 'react-router-dom';

import UserCard from '../index';

describe('<UserCard />', () => {
  test('renders', () => {
    const mockData = {
      name: 'Welly',
      phone: '007',
      email: 'test@gmail.com',
      website: 'www.test.com'
    };
    const tree = renderer
      .create(
        <StaticRouter context={{}}>
          <UserCard info={mockData} />
        </StaticRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
