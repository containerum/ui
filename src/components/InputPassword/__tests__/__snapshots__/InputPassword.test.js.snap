// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserCard /> renders 1`] = `
<input
  className="form-control custom-input"
  disabled={undefined}
  id={undefined}
  onChange={[Function]}
  placeholder="Password"
  type="password"
  value={undefined}
/>
`;
